mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1170,20 +1174,22 @@ def _log_custom_metric_artifact(self, artifact_name, raw_artifact, custom_metric
         artifact._load(artifact_file_local_path)
         return artifact
 
-    def _get_args_for_metrics(self, extra_metric, eval_df):
+    def _get_args_for_metrics(self, metric_tuple, eval_df):
```

Comment:
Can we add return type annotations and docstrings for this, since the return types seem like they can vary?